common/app/views/fragments/footer.scala.html

```diff
@@ -146,7 +146,7 @@
             </a>
             <div class="really-serious-copyright">
                 @if(!page.metadata.isFoundation) {
-                    © @{new DateTime().year.getAsText} Guardian News &amp; Media Limited or its affiliated companies. All rights reserved.
+                    © @{new DateTime().year.getAsText} Guardian News &amp; Media Limited or its affiliated companies. All rights reserved [16].
```

Comment:
```suggestion
                    © @{new DateTime().year.getAsText} Guardian News &amp; Media Limited or its affiliated companies. All rights reserved.
```

Reply:
Yeah, it was part of the review (I promised I would remove it before merging ^^)